kernel/src/net/socket/unix/ns/abs.rs

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use alloc::{collections::btree_map::Entry, format};
+
+use keyable_arc::KeyableArc;
+
+use crate::prelude::*;
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
`derive(Hash)` seems to be useless. We don't have usable hash table yet.

Reply:
Thanks for the information. I didn't even know that `HashSet` and `HashMap` don't live in `alloc`. That surprises me.